lib/rubocop/cop/style/inverse_methods.rb

```diff
@@ -143,6 +147,19 @@ def end_parentheses(node, method_call)
                                     method_call.loc.expression.end_pos,
                                     node.loc.expression.end_pos)
         end
+
+        # When comparing classes, `!(Integer < Numeric)` is not the same as
+        # `Integer > Numeric`.
+        def could_be_check_for_class_hierarchy?(lhs, rhs, method)
+          return false unless CLASS_COMPARISON_METHODS.include?(method)
+          return true if camel_case_constant?(lhs)
```

Comment:
I guess you can just have one big `&&` here. Might read better. At least for me it'd be better.